after_deployment_tasks.md

```diff
@@ -13,6 +13,14 @@ Check that you now
 
 * see "1+2=3" in each app.
 * have been redirected to https
+
+Setup cloudflare: https://www.cloudflare.com/cloudflare-settings?z=renuoapp.ch
```

Comment:
they just changed the url to https://www.cloudflare.com/a/dns/renuoapp.ch i guess. will fix it, thx!